protocol/src/common/protocol.notebook.ts

```diff
@@ -375,6 +375,7 @@ export namespace NotebookDocumentSyncRegistrationType {
 	export const method: 'notebookDocument/sync' = 'notebookDocument/sync';
 	export const messageDirection: MessageDirection = MessageDirection.clientToServer;
 	export const type = new RegistrationType<NotebookDocumentSyncRegistrationOptions>(method);
+	export const capabilities: { client: 'notebookDocument.synchronization'; server: 'notebookDocumentSync' } = { client: 'notebookDocument.synchronization', server: 'notebookDocumentSync' };
```

Comment:
I need some help defining the rest of the notebook capabilities. It isn't very clear from the spec how those are defined.